fragile/core/walkers.py

```diff
@@ -550,9 +550,7 @@ def get_best_index(self) -> int:
         if len(rewards) == 0:
             return self.n - 1
         best = rewards.argmin() if self.minimize else rewards.argmax()
```

Comment:
@justindujardin can you please just change this line for:
```suggestion
        best = rewards.min() if self.minimize else rewards.max()
```

The test is very nice and should be added to the PR, thanks.

Reply:
This should solve the bug and make your test pass.